Scenes/Entities/Crawler/crawler.gd

```diff
@@ -0,0 +1,59 @@
+extends Node2D
+class_name EnemyNode
```

Comment:
Note: I'd make players, NPCs and enemies all inherit some base class, for example Actor. It's a preferential thing but lets me put some basic stuff like damage and other functions in one place.
Also, when you're making a base class (like this one) generally don't give it functionality, instead make a base EnemyNode class and and CrawlerEnemy class that extends it with the functionality. Also
![dockfunni](https://github.com/user-attachments/assets/725a6429-2ad7-4ca5-8b18-8bc755ef5698)